pywbem/_subscription_manager.py

```diff
@@ -779,11 +781,14 @@ def add_filter(self, server_id, source_namespace, query,
 
     def get_owned_filters(self, server_id):
         """
-        Return the owned indication filters in a WBEM server that have been
-        created by this subscription manager.
+        Return the indication filters in a WBEM server that are owned by this
+        subscription manager.
 
         This function accesses only the local list of owned filters; it does
-        not contact the WBEM server.
+        not contact the WBEM server. The local list of owned filters is
+        discovered from the WBEM server when the server is registered with the
+        subscription manager, and is from then on maintained as changes happen
```

Comment:
change "is from then on maintained as" to is maintained from then on"

Reply:
DONE (3 occurrences).